src/plonk/mod.rs

```diff
@@ -66,6 +71,56 @@ pub enum Error {
     },
 }
 
+#[derive(Clone, PartialEq, Serialize, Default)]
+pub struct CompressedCustomGatesLookupView<F: PrimeField> {
```

Comment:
We need a slighly better name, every time I saw this, it takes me a while to realize it is not the view for lookup but the view of CompressedCustomGatesLookup.